### ISSUE TITLE
allow customization of fetcher image url

### ISSUE THREAD

**Issue by Slahser** [fission/fission#184]
thx for your work first.

i am building a fission environment offline

because fetcher block it 

```
fission --server http://ip:31313 env create --name nodejs --image registry.me.com/node-env
```

is there any flag to solve it?

**Comment by soamvasani** [comment]
This isn't configurable yet.  

If anyone would like to take a crack at fixing this before I get to it, this is the place where we have the fetcher URL:
https://github.com/fission/fission/blob/master/poolmgr/gp.go#L355

We should change that to an env var, and then the env var could be set through the deployment yaml.

**Comment by Slahser** [comment]
sigh for my poor golang ability...

waiting for a good pr

**Comment by soamvasani** [comment]
Available since 0.2.1 (specify `fetcherImage` and `fetcherImageTag` as helm variables).
